docs/advanced/input_files/input-main.md

```diff
@@ -107,20 +89,22 @@ This part of variables are used to control general system parameters.
 
 - **Type**: String
 - **Description**: Specify the type of calculation.
+
   - *scf*: do self-consistent electronic structure calculation
   - *relax*: do structure relaxation calculation, one can ues `relax_nmax` to decide how many ionic relaxations you want.
   - *cell-relax*: do variable-cell relaxation calculation.
   - *nscf*: do the non self-consistent electronic structure calculations. For this option, you need a charge density file. For nscf calculations with planewave basis set, pw_diag_thr should be <= 1d-3.
   - *istate*: For LCAO basis. Please see the explanation for variable `nbands_istate`.
   - *ienvelope*: Envelope function for LCAO basis. Please see the explanation for variable `nbands_istate`.
   - *md*: molecular dynamics
+  - *ofdft*: do orbital-free dft calculation.
```

Comment:
Why did you name this calculation : "of-scf" , which is like "sto-scf"?

Reply:
Sorry for replying so late. I prefer `ofdft` because the main process of OFDFT is to solve an optimization problem, which is quite different from the SCF of KSDFT or sto-DFT.